lib/model/parameter/SortBy.dart

```diff
@@ -27,3 +20,18 @@ enum SortOption {
   CREATED,
   EDIT,
 }
+
+extension SortOptionExt on SortOption {
+  String get name {
+    switch (this) {
+      case SortOption.PRODUCT_NAME:
+        return 'product_name';
+      case SortOption.CREATED:
+        return 'created_t';
+      case SortOption.EDIT:
+        return 'last_modified_t';
+      case SortOption.POPULARITY:
+        return 'unique_scans_n';
```

Comment:
There's a sort by nothing option
https://github.com/openfoodfacts/openfoodfacts-server/pull/5779

Reply:
Hello!
Do you want me to add it in this pull request?

Reply:
Done